test/e2e/cypress/e2e/users.cy.js

```diff
@@ -0,0 +1,326 @@
+    it('should show changed by other user warning', () => {
+      const { fullname: fullname1 } = userFactory.build();
+
+      getUserIdFromPath().then((id) => patchUser(id, { fullname: fullname1 }));
+
+      const { fullname: fullname2 } = userFactory.build();
+      cy.get('input[placeholder="Enter full name"]').clear();
+      cy.get('input[placeholder="Enter full name"]').type(fullname2);
+      cy.contains('button', 'Save').click();
+
+      cy.get('span').contains('Information has been updated by another user');
+      cy.reload();
+      cy.contains('Information has been updated by another user').should(
+        'not.exist'
+      );
+    });
+
+    it('should edit full name properly', () => {
+      const { fullname } = userFactory.build();
+      cy.get('input[placeholder="Enter full name"]').clear();
+      cy.get('input[placeholder="Enter full name"]').type(fullname);
+      cy.contains('button', 'Save').click();
+      cy.get('div').contains('User edited successfully');
+      cy.get('h1').should('contain', 'Users');
+      cy.get('p').contains(fullname);
+    });
+  });
+
+  describe('User profile', () => {
+    before(() => {
+      cy.logout();
+    });
+
+    it('should login with the new user', () => {
+      cy.login(USER.username, PASSWORD);
+      cy.visit('/');
+      cy.url().should('include', '/');
+      cy.get('button').contains(USER.username);
+    });
+
+    it('should not see Users entry in the sidebar', () => {
+      cy.contains('Users').should('not.exist');
+    });
+
+    it('should get a forbidden messages for user related pages', () => {
+      cy.visit('/users');
+      cy.get('div').contains('Access to this page is forbidden');
+    });
+
+    it('should see password change suggestion toast', () => {
+      cy.get('p').contains('Password change is recommended.');
+    });
+
+    it('should have a proper user data in the profile view', () => {
+      cy.contains('a', 'Profile').click();
+      cy.get('input[placeholder="Enter email address"]').should(
+        'have.value',
+        USER.email
+      );
+      cy.get('input[aria-label="username"]').should(
+        'have.value',
+        USER.username
+      );
+    });
+
+    it('should edit full name properly from the profile view', () => {
+      const { fullname } = userFactory.build();
+      cy.get('input[placeholder="Enter full name"]').clear();
+      cy.get('input[placeholder="Enter full name"]').type(fullname);
+      cy.contains('button', 'Save').click();
+      cy.get('div').contains('Profile changes saved!');
+    });
+
+    it('should fail editing user password if current password is wrong', () => {
+      cy.contains('button', 'Change Password').click();
+      cy.get('input').eq(4).type('wrong');
+      cy.get('input').eq(5).type(PASSWORD);
+      cy.get('input').eq(6).type(PASSWORD);
+      cy.get('button:contains("Save")').eq(1).click();
+      cy.get('p').contains('Is invalid');
+    });
+
+    it('should edit the user password', () => {
```

Comment:
We didn't include this scenario in our "test list". 
I guess there are many many more scenarios that we don't test.
Anyway, since you want this one specifically, I will add it hehe

Reply:
@EMaksy I couple of new tests added.
Admin user cannot be edited in the users view and admin cannot be edited in the profile view.